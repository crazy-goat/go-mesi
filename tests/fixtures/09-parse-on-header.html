<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
Recursive parse 1 response: [<esi:include src="http://127.0.0.1:8080/returnEsi" />]
Should return no parsed content: [<esi:include src="http://127.0.0.1:8080/returnNonEsiHeader" />]
</body>
</html>
