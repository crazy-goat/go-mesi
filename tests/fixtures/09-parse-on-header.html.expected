<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
Recursive parse 1 response: [included: [Hello World]]
Should return no parsed content: [included: [<esi:include src="http://127.0.0.1:8080/hello" />]]
</body>
</html>
